File: progra3-0583-equipo5-frontend/Frontend_C#/SoftPac/SoftPacPersistance/DAO/PaisesDAO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using SoftPac.Model;

namespace SoftPac.Persistance.DAO
{
    public interface PaisesDAO
    {
        int Insertar(PaisesDTO pais);
        int Modificar(PaisesDTO pais);
        int Eliminar(PaisesDTO pais);
        PaisesDTO ObtenerPorId(int paisId);
        IList<PaisesDTO> ListarTodos();
    }
}

File: BackEnd/Backend_Java/SoftPacPersistance/src/main/java/pe/edu/pucp/softpac/dao/PaisesDAO.java
package pe.edu.pucp.softpac.dao;

import pe.edu.pucp.softpac.model.PaisesDTO;

import java.util.ArrayList;
import java.util.List;

public interface PaisesDAO {
    public Integer insertar(PaisesDTO pais);

    public PaisesDTO obtenerPorId(Integer paisId);

    public List<PaisesDTO> listarTodos();

    public Integer modificar(PaisesDTO pais);

    public Integer eliminar(PaisesDTO pais);
}
